src/endpoints/clip/clip.endpoint.ts

```diff
@@ -0,0 +1,40 @@
+import { cast } from '@deepkit/type'
+import { BaseEndpoint } from '../endpoint.base'
+import type { DownloadClipResponse } from './dto/download-clip.response'
+import type { GetClipResponse } from './dto/get-clip.response'
```

Comment:
The `import type` statement unfortunately doesn't work with `@deepkit/type` as it needs the type information at runtime. We'll need to drop the `type` statement from the import and disable eslint for the follow line as it'll complain as it's only used in the generic type parameter for `cast`.

See [channel.instance.ts](https://github.com/zSoulweaver/kient/blob/master/src/endpoints/channel/instance/channel.instance.ts) for an example here.